gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/management-openapi-v2.yaml

```diff
@@ -1617,6 +1617,27 @@ paths:
                 default:
                     $ref: "#/components/responses/Error"
 
+    # API Documentation
+    /environments/{envId}/apis/{apiId}/pages:
+        parameters:
+            - $ref: "#/components/parameters/envIdParam"
+            - $ref: "#/components/parameters/apiIdParam"
+        get:
+            tags:
+                - API Documentation
+            summary: Get API documentation pages
+            description: |-
+                Get API documentation pages. The result contains a list of pages and folders of the API.
```

Comment:
**question**: How folders and subfolders are going to be retrieved?
All documents, whatever in which folder they are?
Or first "Home" and then you use a path or query parameter to fetch subfolder content?

Reply:
I was thinking about adding a query parameter to specify the parent folder. When no parent specified : you get the pages and folders in the root folder. With a parameter specified : you get all pages and folders in the specified folder. 

It's not done in this PR. It will be added in future ones.

Reply:
👍🏻 I would prefer a queryParam with `/` as default value instead of "no param means home", because usually no param means "no filter" i.e. everything.